interfaces/apparmor/template.go

```diff
@@ -520,144 +455,22 @@ var defaultCoreRuntimeTemplateRules = `
 
   # for bash 'binaries' (do *not* use abstractions/bash)
   # user-specific bash files
-  /{,usr/}bin/bash ixr,
-  /{,usr/}bin/dash ixr,
+  /{,usr/}bin/{bash,dash} ixr,
   /usr/share/terminfo/** r,
 
   # Common utilities for shell scripts
-  /{,usr/}bin/arch ixr,
-  /{,usr/}bin/{,g,m}awk ixr,
-  /{,usr/}bin/base32 ixr,
-  /{,usr/}bin/base64 ixr,
-  /{,usr/}bin/basename ixr,
-  /{,usr/}bin/bunzip2 ixr,
-  /{,usr/}bin/busctl ixr,
-  /{,usr/}bin/bzcat ixr,
-  /{,usr/}bin/bzdiff ixr,
-  /{,usr/}bin/bzgrep ixr,
-  /{,usr/}bin/bzip2 ixr,
-  /{,usr/}bin/cat ixr,
-  /{,usr/}bin/chgrp ixr,
-  /{,usr/}bin/chmod ixr,
-  /{,usr/}bin/chown ixr,
-  /{,usr/}bin/clear ixr,
-  /{,usr/}bin/cmp ixr,
-  /{,usr/}bin/cp ixr,
-  /{,usr/}bin/cpio ixr,
-  /{,usr/}bin/cut ixr,
-  /{,usr/}bin/date ixr,
-  /{,usr/}bin/dbus-daemon ixr,
-  /{,usr/}bin/dbus-run-session ixr,
-  /{,usr/}bin/dbus-send ixr,
-  /{,usr/}bin/dd ixr,
-  /{,usr/}bin/diff{,3} ixr,
-  /{,usr/}bin/dir ixr,
-  /{,usr/}bin/dirname ixr,
-  /{,usr/}bin/du ixr,
-  /{,usr/}bin/echo ixr,
-  /{,usr/}bin/{,e,f,r}grep ixr,
-  /{,usr/}bin/env ixr,
-  /{,usr/}bin/expr ixr,
-  /{,usr/}bin/false ixr,
-  /{,usr/}bin/find ixr,
-  /{,usr/}bin/flock ixr,
-  /{,usr/}bin/fmt ixr,
-  /{,usr/}bin/fold ixr,
-  /{,usr/}bin/getconf ixr,
-  /{,usr/}bin/getent ixr,
-  /{,usr/}bin/getopt ixr,
-  /{,usr/}bin/groups ixr,
-  /{,usr/}bin/gzip ixr,
-  /{,usr/}bin/head ixr,
-  /{,usr/}bin/hostname ixr,
-  /{,usr/}bin/id ixr,
-  /{,usr/}bin/igawk ixr,
-  /{,usr/}bin/infocmp ixr,
-  /{,usr/}bin/kill ixr,
-  /{,usr/}bin/ldd ixr,
+  /{,usr/}bin/{arch,{,g,m}awk,base{32,64,name},bunzip2,busctl,bz{cat,diff,grep,ip2},cat,ch{grp,mod,own},clear,cmp,cp,cpio,cut,date,dbus-{daemon,run-session,send},dd,diff{,3},dir{,name},du,echo,{,e,f,r}grep,env,expr,false,find,flock,fmt,fold,get{conf,ent,opt},groups,gzip,head,hostname,id,igawk,infocmp,kill,ldd,less{,file,pipe},ln,line,link,locale,logger,ls,md5sum,mk{dir,fifo,nod,temp},more,mv,nice,nohup,numfmt,od,openssl,paste,pgrep,print{env,f},ps,pwd,rea{dlink,lpath},rev,rm{,dir},run-parts,sed,seq,sha{1,224,256,384,512}sum,shuf,sleep,sort,stat,stdbuf,stty,sync,systemd-cat,tac,tail,tar,tee,test,tempfile,tset,touch,tput,tr,true,tty,uname,un{iq,link,xz,zip},uptime,vdir,wc,which{,.debianutils},xargs,xz,yes,zcat,z{,e,f}grep,zip{,grep}} ixr,
```

Comment:
this is pretty ugly - could we perhaps remove the simplifications within the various binary names - ie. just something like the following so that is easier to read:

```
/{,usr/}bin/{arch,awk,gawk,mawk,...} ixr
```

Reply:
yeah, I was wondering about this. I needed extra opinion, I changed it now to prefer readability in this case